### DISCUSSION TITLE
如何让docker容器支持pandas

### DISCUSSION THREAD

**Discussion by sawyerYoung** [QingdaoU/OnlineJudge#467] [Q&A] [answered]
如果在代码中加入import pandas 会引发 runtime error，在JudgeServer讨论区4年前有人发问了，但是没结果，求解答

**Comment by mikucat0309** [comment, marked as answer]
默认的 judge server 容器映像档没有安装标准函式库以外的函式库，有需要可以自行建置。
新版文档有写，不过目前只写了英文版的哈 😅
https://opensource.qduoj.com/#/en/guide/judge_environment
